src/QsCompiler/CompilerOptimizations/Utils.fs

```diff
@@ -41,12 +67,25 @@ let rec internal toSymbolTuple (x: QsTuple<LocalVariableDeclaration<QsLocalSymbo
     | QsTuple items when items.Length = 1 ->
         toSymbolTuple items.[0]
     | QsTuple items ->
-        VariableNameTuple ((Seq.map toSymbolTuple items).ToImmutableArray())
+        Seq.map toSymbolTuple items |> ImmutableArray.CreateRange |> VariableNameTuple
+
+/// Matches a TypedExpression as a SymbolTuple
```

Comment:
It's probably worth mentioning what the default case is.

Reply:
What do you mean? I don't think this match statement has a default case.

Reply:
The default case is just anything that's not a SymbolTuple

Reply:
Yes, but it would be nice to point out in the doc comment.